lib/common/session_model.dart

```diff
@@ -546,9 +554,7 @@ class SessionModel extends Model {
   Future<void> reportIssue(
       String email, String issue, String description) async {
     if (isDesktop()) {
-      await ffiReportIssue(email.toNativeUtf8(), issue.toNativeUtf8(),
-          description.toNativeUtf8());
-      return;
+      return await compute(ffiReportIssue, [email, issue, description]);
```

Comment:
I added compute, without this, the UI was freezing until the call completes. I think we need need to do other places as well.